templates/careers/hiring-process/index.html

```diff
@@ -31,14 +31,14 @@ <h1>Canonical's hiring process</h1>
     </div>
     <div class="u-fixed-width">
       <div class="p-image-container--cinematic is-cover u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/9acad1b0-department-sales-%402x.jpg",
-                alt="",
-                width="2216",
-                height="834",
-                hi_def=True,
-                loading="auto",
-                attrs={"class": "p-image-container__image"},) | safe
-        }}
+        <iframe width="1120"
```

Comment:
Needed for the utility to work
```suggestion
        <iframe width="1120" class="u-embedded-media__element"
       
```